stack-orchestration/configuration/di-id-reuse-core-build-admin/ais-test-ecr/parameters.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "ParameterKey": "AWSOrganizationId",
+    "ParameterValue": "o-dpp53lco28"
+  },
+  {
+    "ParameterKey": "PipelineStackName",
+    "ParameterValue": "ais-main-pipeline"
```

Comment:
This value should not be same as main stack, possibly `ais-test-ecr`.

Reply:
Hey @TaherKapasi  so reason for this being `ais-main-pipeline` is that it needs to be the name of the pipeline stack that connects to GitHub Actions and for this we used `ais-main-pipeline` instead as its what is creating our `GitHubActionsRoleArn`